exercises/practice/resistor-color-duo/.meta/config.json

```
@@ -1,6 +1,13 @@
 {
   "blurb": "Convert color codes, as used on resistors, to a numeric value.",
-  "authors": [],
+  "authors": [
+    "amscotti"
+  ],
+  "contributors": [
+    "devkabiir",
+    "ErikSchierboom",
+    "Stargator"
+  ],
```

```suggestion
  "contributors": [
  ],
```

---

Adding my name to the contributors due to the revision of the exercise to sync it with the spec

```suggestion
  "contributors": [
    "Stargator"
  ],
```